AirSkyBoat/AirSkyBoat
# moved toraimarai turmoil to new quest format

**_I affirm:_**

## What does this pull request do?

Closes #1779 
Water way to go should no longer block toraimai trouble. note that because both quests are repeatable, you may get the "repeat" dialog for water way to go before the opening toraimai cs. should this happen, just talk to the quest giver again and you'll get the cs for this quest

## Steps to test these changes

1. Complete Water Way To Go
2. Zone out of windy waters
3. zone back into waters and talk to Ohbiru-Dohbiru
4. opening CS for toraimarai turmoil should play, and you should be able to complete the quest, as well as repeat it. note that you may need to talk to him twice, since water way to go is repeatable, you may get the repeat quest message.
